>>> 'Pkg.add("CauseMap")' log
INFO: Cloning cache of CauseMap from https://github.com/cyrusmaher/CauseMap.jl.git
INFO: Installing BinDeps v0.6.0
INFO: Installing CauseMap v0.0.3
INFO: Installing Conda v0.7.0
INFO: Installing MacroTools v0.3.7
INFO: Installing PyCall v1.14.0
INFO: Installing SHA v0.5.1
INFO: Installing URIParser v0.2.0
INFO: Building Conda
INFO: Building PyCall
INFO: Using the Python distribution in the Conda package by default.
To use a different Python version, set ENV["PYTHON"]="pythoncommand" and re-run Pkg.build("PyCall").
INFO: PyCall is using /home/vagrant/.julia/v0.6/Conda/deps/usr/bin/python (Python 2.7.13) at /home/vagrant/.julia/v0.6/Conda/deps/usr/bin/python, libpython = /home/vagrant/.julia/v0.6/Conda/deps/usr/lib/libpython2.7
INFO: /home/vagrant/.julia/v0.6/PyCall/deps/deps.jl has not changed
INFO: /home/vagrant/.julia/v0.6/PyCall/deps/PYTHON has not changed
INFO: Package database updated
INFO: METADATA is out-of-date â€” you may not have the latest version of CauseMap
INFO: Use `Pkg.update()` to get the latest versions of your packages

>>> 'Pkg.test("CauseMap")' log
Julia Version 0.6.0
Commit 9036443 (2017-06-19 13:05 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz
  WORD_SIZE: 64
           Ubuntu 14.04.5 LTS
  uname: Linux 3.13.0-128-generic #177-Ubuntu SMP Tue Aug 8 11:40:23 UTC 2017 x86_64 x86_64
Memory: 2.9392738342285156 GB (1069.44921875 MB free)
Uptime: 21908.0 sec
Load Avg:  0.99267578125  0.9833984375  0.79296875
Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz: 
       speed         user         nice          sys         idle          irq
#1  3500 MHz    1147122 s       6583 s      61673 s     689079 s         25 s
#2  3500 MHz     329888 s         21 s      29128 s    1781372 s          0 s

  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, haswell)
Environment:
  TERM = vt100
  LD_LIBRARY_PATH = :/usr/local/lib/
  PATH = /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/vagrant/julia/bin/
  JAVA_HOME = /usr/lib/jvm/java-8-openjdk-amd64
  HOME = /home/vagrant

Package Directory: /home/vagrant/.julia/v0.6
2 required packages:
 - CauseMap                      0.0.3
 - JSON                          0.13.0
7 additional packages:
 - BinDeps                       0.6.0
 - Compat                        0.31.0
 - Conda                         0.7.0
 - MacroTools                    0.3.7
 - PyCall                        1.14.0
 - SHA                           0.5.1
 - URIParser                     0.2.0
INFO: Testing CauseMap
ERROR: LoadError: LoadError: LoadError: LoadError: UndefVarError: ASCIIString not defined
Stacktrace:
 [1] include_from_node1(::String) at ./loading.jl:569
 [2] include(::String) at ./sysimg.jl:14
 [3] include_from_node1(::String) at ./loading.jl:569
while loading /home/vagrant/.julia/v0.6/CauseMap/src/plotting.jl, in expression starting on line 1
while loading /home/vagrant/.julia/v0.6/CauseMap/src/CauseMap.jl, in expression starting on line 5
while loading /home/vagrant/.julia/v0.6/CauseMap/test/testplotting.jl, in expression starting on line 1
while loading /home/vagrant/.julia/v0.6/CauseMap/test/runtests.jl, in expression starting on line 1
==============================[ ERROR: CauseMap ]===============================

failed process: Process(`/home/vagrant/julia/bin/julia -Cx86-64 -J/home/vagrant/julia/lib/julia/sys.so --compile=yes --depwarn=yes --check-bounds=yes --code-coverage=none --color=no --compilecache=yes /home/vagrant/.julia/v0.6/CauseMap/test/runtests.jl`, ProcessExited(1)) [1]

================================================================================
ERROR: CauseMap had test errors

>>> End of log
